Changes needed to get serf to build on Solaris. Taken from:
http://mail-archives.apache.org/mod_mbox/subversion-dev/201411.mbox/<87ppci4184.fsf@ntlworld.com>

These changes are already known upstream.

--- serf-1.3.8/SConstruct.orig	2015-03-25 06:42:50.687989565 -0700
+++ serf-1.3.8/SConstruct	2015-03-25 06:44:01.084907971 -0700
@@ -210,7 +210,7 @@
 # Unfortunately we can't set the .dylib compatibility_version option separately
 # from current_version, so don't use the PATCH level to avoid that build and
 # runtime patch levels have to be identical.
-env['SHLIBVERSION'] = '%d.%d.%d' % (MAJOR, MINOR, 0)
+#env['SHLIBVERSION'] = '%d.%d.%d' % (MAJOR, MINOR, 0)
 
 LIBNAME = 'libserf-%d' % (MAJOR,)
 if sys.platform != 'win32':
@@ -248,6 +248,7 @@
 
   if sys.platform == 'sunos5':
     env.Append(LIBS='m')
+    env['PLATFORM'] = 'posix'
 else:
   # Warning level 4, no unused argument warnings
   env.Append(CCFLAGS=['/W4', '/wd4100'])
